src/routes/blog/(articles)/[article]/+page.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+	export let data;
+	import '$lib/articles.scss';
+	import 'highlight.js/styles/github.css';
+
+	import { Image } from '@unpic/svelte';
+	import dayjs from 'dayjs';
+	import hljs from 'highlight.js';
+	import { onMount } from 'svelte';
+
+	import Link from '$lib/components/Link.svelte';
+	import { sanitizeTiptapContent } from '$lib/utils/sanitizeTiptapContent.js';
+	onMount(() => {
+		hljs.highlightAll();
```

Comment:
https://www.npmjs.com/package/svelte-highlight

Reply:
Im confused what benefits this has over just use highlight js

Reply:
nvm